src/translib/transformer/xfmr_vrf.go

```diff
@@ -0,0 +1,517 @@
+package transformer
+
+import (
+        "errors"
+        log "github.com/golang/glog"
+        "github.com/openconfig/ygot/ygot"
+        "strings"
+        "translib/ocbinds"
+)
+
+type NwInstMapKey struct {
+        NwInstName    string
+        NwInstType    string
+}
+
+const (
+        MGMT_VRF_ENABLE        = "mgmtVrfEnabled"
+)
+
+const (
+        MGMT_VRF_NAME          = "mgmt-vrf-name"
+)
+
+var nwInstTypeMap = map[ocbinds.E_OpenconfigNetworkInstanceTypes_NETWORK_INSTANCE_TYPE] string {
+        ocbinds.OpenconfigNetworkInstanceTypes_NETWORK_INSTANCE_TYPE_UNSET: "",
+        ocbinds.OpenconfigNetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE: "DEFAULT_INSTANCE",
+        ocbinds.OpenconfigNetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L2L3: "L2L3",
+        ocbinds.OpenconfigNetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L2P2P: "L2P2P",
+        ocbinds.OpenconfigNetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L2VSI: "L2VSI",
```

Comment:
Not sure why we need to add all instance types here, why cant we keep only the required inst types.

Reply:
the set of fields are in config & state for the network instance openconfig yang. From talking to Kwan, need field transformer for each, and returning empty map for now.